fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2/lang/en_us.json

```diff
@@ -168,6 +168,7 @@
   "tag.item.c.ender_pearls": "Ender Pearls",
   "tag.item.c.fertilizers": "Fertilizers",
   "tag.item.c.foods": "Foods",
+  "tag.item.c.animal_foods": "Animal Foods",
```

Comment:
You havent added this to the language builder.

Reply:
I have pushed the fix.

Reply:
Oh thank you, you beat me to it